cephprimarystorage/cephprimarystorage/cephagent.py

```
@@ -691,6 +691,15 @@ def get_facts(self, req):
         rsp.type = ceph.get_ceph_manufacturer()
 
         ip_addresses = [chunk.address for chunk in [x for x in iproute.query_addresses(ip_version=4) if x.address != '127.0.0.1' and not x.ifname.endswith('zs')]]
+        # IPv6 支持：使用 ipaddress 标准库过滤链路本地地址（RFC 4291 fe80::/10）
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)